syz-verifier/stats.go

```diff
@@ -129,19 +128,10 @@ func (s *Stats) getOrderedStats() []*CallStats {
 
 func (s *Stats) getOrderedStates(call string) []string {
 	states := s.Calls[call].States
-	ss := make([]int, 0, len(states))
+	ss := make([]string, 0, len(states))
 	for s := range states {
-		ss = append(ss, s)
+		ss = append(ss, fmt.Sprintf("%q", s.String()))
```

Comment:
This function can be a bit simpler as:
```
	var ss []string
	for s := range s.Calls[call].States {
		ss = append(ss, s.String())
	}
	sort.Strings(ss)
	return ss
```

Reply:
My intention was to have each state inside quotation marks so there is a clear delimitation.

Reply:
They will be printed as `["Flags: 1, Errno:2" "Flags:7, Errno:13"]`, does this look fine to you? Also removed `.String()`, I was not aware it's not needed.

Reply:
I think it's fine either way, I don't think these [] are important.